From 80806f5438c769b71e92a7669db029f5f243cfce Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 12 Apr 2024 22:06:55 +0100
Subject: [PATCH] po-docs: Remove virt-dib from list of translated man pages

This page no longer exists.

Fixes: commit 57423d907270526ea664ff15601cce956353820e
(cherry picked from commit 2e797d3bb4c8a13305a948e3d87e4eabfd692e17)
---
 po-docs/language.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/po-docs/language.mk b/po-docs/language.mk
index ba2e36998..7328265e6 100644
--- a/po-docs/language.mk
+++ b/po-docs/language.mk
@@ -30,7 +30,6 @@ MANPAGES = \
 	virt-cat.1 \
 	virt-customize.1 \
 	virt-df.1 \
-	virt-dib.1 \
 	virt-diff.1 \
 	virt-edit.1 \
 	virt-filesystems.1 \
